Samples.xcodeproj/project.pbxproj

```diff
@@ -1222,7 +1248,7 @@
 			repositoryURL = "https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/";
 			requirement = {
 				kind = upToNextMinorVersion;
-				minimumVersion = 200.0.0-beta;
+				minimumVersion = "200.0.0-beta";
```

Comment:
```suggestion
				minimumVersion = 200.0.0-beta;
```

Reply:
It's fine to have this. I think it may be Xcode upgrade changing the behavior of quotation mark usage.

Reply:
ok, I added them back in